cmd/cluster/powervs/create.go

```diff
@@ -139,6 +139,7 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 			PowerVSCloudConnection: opts.PowerVSPlatform.PowerVSCloudConnection,
```

Comment:
please conflate this commit with https://github.com/openshift/hypershift/pull/1225/commits/e10d67b93e7d4e08939c34d1cd53c4c0c7c463d0